### DISCUSSION TITLE
Settings made with EP on Win11 22623 restored after upgrading to newest Dev Channel Build

### DISCUSSION THREAD

**Discussion by Hibiki0506** [valinet/ExplorerPatcher#1357] [General]
Hello,

I have a problem with upgrading to a newer build in the Dev Channel.

I have uninstalled EP some time ago, where I've preferred the Win10 Ribbon menu instead of the Win11 icons in the Explorer.
Since I have uninstalled EP, I thought that the settings would be deleted too, but every time I am trying to get to a new Dev Beta, Windows restores the Win10 Ribbon menu, even with EP not installed. And I have no clue, why this happens or how I could switch the Ribbon back to the Win11 icon menu.

And it seems that even the setting for the old context menu is restored after updating to a Dev Channel build.

Does someone have the same problem?

Edit: In build 23440, right-clicking in the explorer crashes the explorer, since the old right-click menu is set by the old EP settings.

**Comment by Hibiki0506** [comment]
![image](https://user-images.githubusercontent.com/32351488/233794366-20f2a15c-f48e-4b26-8ed2-a5f2d5b44258.png)
And here a image of the explorer window with the Win10 ribbon menu. EP is not installed.

**Comment by pyrates999** [comment]
dev channel and insider builds are not supported due to symbols not being available for them.

**Comment by Hibiki0506** [comment]
I know, that's why I said that EP is not installed anymore. The settings, which should not exist anymore, since EP is not installed anymore for a long while now, are being restored by Windows while upgrading to a newer version. It seems that the uninstaller isn't working properly, if it can't delete its own files. But that is something which would not be fixed, when I look into other discussions or issues.
